nx-dev/nx-dev/public/documentation/latest/react/guides/using-tailwind-css-in-react.md

```diff
@@ -46,7 +46,11 @@ Nx has a utility function for determining the glob representation of all files t
 const { createGlobPatternsForDependencies } = require('@nrwl/react/tailwind');
 
 module.exports = {
-  purge: createGlobPatternsForDependencies(__dirname),
+  purge: [
+    // place your own app's glob pattern (for example)
+    './apps/{your app here}/src/**/*.{js,ts,jsx,tsx}',
```

Comment:
I always use:

```ts
join(__dirname, 'THE GLOB')
```

So that it is never app specific and therefore copy pastable between apps.

The same goes for reference tailwind.config.js from postcss.config.js

Reply:
indeed! Good point, thx :)

let me fix that in the docs as well